app/views/errors/invalid_session.en.html.erb

```diff
@@ -0,0 +1,19 @@
+<% title 'We signed you out' %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      For your security, we signed you out
+    </h1>
+
+    <p class="govuk-body">
+      This is because you were inactive for <%= hours_or_minutes(Rails.configuration.x.session.timeout_in) %>.
```

Comment:
Will this timeout_in value always be available? Is there any chance of it being nil and causing an error? i.e. do we need a fallback?

Reply:
It is always available, and there is already a fallback in case the ENV is not defined:
```ruby
config.x.session.timeout_in = ENV.fetch('SESSION_TIMEOUT_MINUTES', 60).to_i.minutes
```